server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -59,6 +60,10 @@ function getRoomId(room: IRoom) {
 	return `${room.tenantId}/${room.documentId}`;
 }
 
+function getClientRoomId(clientId: string) {
```

Comment:
can we use this method on line 379 as well, where we set the client to join its room?